WARNING:root:this is my warning log
ERROR:root:this is my error
WARNING:root:this is my warning log
ERROR:root:this is my error
WARNING:root:this is my warning log
ERROR:root:this is my error
ERROR:root:Error hs happend
ERROR:root:Exception occureddivision by zero
Traceback (most recent call last):
  File "C:\Users\sinch\AppData\Local\Temp\ipykernel_24108\3222558361.py", line 4, in divbyzero
    div = a/b
          ~^~
ZeroDivisionError: division by zero
INFO:root:this is my info log
WARNING:root:this is my warning log
ERROR:root:this is my error
INFO:root:this is my info log
WARNING:root:this is my warning log
ERROR:root:this is my error
INFO:root:this is a start of my code and i am try to enter 4 and 5
INFO:root:executed succ
INFO:root:this is a start of my code and i am try to enter 4 and 0
ERROR:root:Error hs happend
ERROR:root:Exception occureddivision by zero
Traceback (most recent call last):
  File "C:\Users\sinch\AppData\Local\Temp\ipykernel_15564\3222558361.py", line 4, in divbyzero
    div = a/b
          ~^~
ZeroDivisionError: division by zero
INFO:root:this is a start of my code and i am try to enter 9 and 3
INFO:root:executed succ
